readme.txt

```diff
@@ -1,8 +1,8 @@
 === Plugin Name ===
 Contributors: mboynes, alleyinteractive
 Tags: editing, duplication, replacement, workflow
-Requires at least: 3.5
-Tested up to: 3.6
+Requires at least: 4.9
```

Comment:
Is there any reason why we cannot support back to 3.5?

Reply:
Ops!